core/templates/dev/head/services/stateful/BackgroundMaskService.js

```diff
@@ -0,0 +1,16 @@
+// Service for enabling a background mask that leaves navigation visible.
```

Comment:
When we make a new file we include the licensing info. For example, please have look at file `TrainingModalService.js` in 'files changed' tab [here](https://github.com/oppia/oppia/pull/3920). Also notice we have a @fileoverview comment under it giving a overview of what that file contains. Could you please format this file in a similar way?